lib/middleware/require-github-user.js

```diff
@@ -11,7 +11,12 @@ module.exports = async function requireGitHubUser(req, res, next) {
   const { slackUser } = res.locals;
 
   if (!slackUser.GitHubUser) {
-    const state = new SignedParams({ slackAction: { ...req.body, token: undefined } });
```

Comment:
What was the reason to have `token: undefined` and why it is no longer needed?

Reply:
This was solely a measure to not leak the `token` (which is part of `req.body`) 🙂

Reply:
I like making sure, it is definitely not leaked from a security perspective. However I would prefer using `null` whenever an assignment to "nothing" is intended, so that nobody confuses that with the value not being assigned at all. But that's just opinion and at most something to consider in the future.